pkgs/applications/graphics/xournalpp/default.nix

```diff
@@ -22,31 +22,20 @@
 # Plugins don't appear to be working in this version, so disable them by not
```

Comment:
Could you remove this comment?

Reply:
I did that and merged.  The last report doesn't show any worsening, so I didn't see any reason for waiting further with this.